oss/sdks/ts/vector/commands/upsert.mdx

```diff
@@ -80,4 +99,53 @@ await index.upsert({
 })
 ```
 
+```typescript Single Data
+await index.upsert({
+  id: "1234",
+  data: "'The Lord of the Rings' follows Frodo Baggins and his allies on a quest to destroy a powerful ring and save Middle-earth from the dark lord Sauron.",
+  metadata: {
+    title: "Lord of The Rings",
+    genre: "drama",
+    category: "classic",
+  },
+});
+```
+
+```typescript Multiple Data
+await index.upsert([
+  {
+    id: "6789",
+    vector:
+      "'Harry Potter' follows the journey of a young wizard, Harry Potter, as he attends Hogwarts School of Witchcraft and Wizardry, forms deep friendships, and confronts the dark wizard Voldemort, who seeks immortality and domination over the magical world.",
+  },
+  {
+    id: "1234",
+    vector:
+      "'The Lord of the Rings' follows Frodo Baggins and his allies on a quest to destroy a powerful ring and save Middle-earth from the dark lord Sauron.",
+    metadata: {
+      title: "Lord of The Rings",
+      genre: "drama",
+      category: "classic",
+    },
+  },
+]);
+```
+
+```typescript Update data
+await index.upsert({
+	id: "1234",
+	vector: "Upstash product"
+	metadata: {
+		title: "Redis"
+	}
+})
+
+await index.upsert({
```

Comment:
what does this command do? There is no data or vector so I didn't get it

Reply:
It basicly updates the metadata